scripts/get_invest_binaries_gcs.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * 2020-07-30 - the workbench build process no longer depends on
+ * pre-built binaries, instead building them itself, so this script
+ * is not being used anymore.
+ * 
+ * This is a utility script for fetching prebuilt invest binaries
+ * that include the server.py module and thus are compatible with the
```

Comment:
Is it worth keeping this file around, or can we just pull it back out from version control if we need it in the future?

Reply:
We could untrack it. That probably is the right thing to do at this point.